packages/gestalt/src/Contents.css

```diff
@@ -26,7 +26,8 @@
 .innerContents {
   composes: flex from "./Layout.css";
   composes: overflowAuto from "./Layout.css";
-  composes: rounding2 from "./Borders.css";
+  position: relative;
+  z-index: 1;
```

Comment:
@ldubberstein thank you for working through this. In Gestalt we want to avoid using `z-index` as much as possible. Instead of applying the `z-index`, could we re-order the `<Caret />` to come before the content? That way the content will overlap the Caret

```diff
diff --git a/packages/gestalt/src/Contents.js b/packages/gestalt/src/Contents.js
index 7e9c191..f41f824 100644
--- a/packages/gestalt/src/Contents.js
+++ b/packages/gestalt/src/Contents.js
@@ -518,6 +518,14 @@ export default class Contents extends React.Component<Props, State> {
           ref={this.setFlyoutRef}
           tabIndex={-1}
         >
+          {caret && (
+            <div
+              className={classnames(colors[bgColor], styles.caret)}
+              style={{ ...caretOffset }}
+            >
+              <Caret direction={mainDir} />
+            </div>
+          )}
           <div
             className={classnames(
               styles.innerContents,
@@ -528,14 +536,6 @@ export default class Contents extends React.Component<Props, State> {
           >
             {children}
           </div>
-          {caret && (
-            <div
-              className={classnames(colors[bgColor], styles.caret)}
-              style={{ ...caretOffset }}
-            >
-              <Caret direction={mainDir} />
-            </div>
-          )}
         </div>
       </div>
     );
```

Also created this video explaining what it looks like in the DOM: https://youtu.be/_4eu_diNopc

Reply:
Good to know!  Yes that works.  I made this change, it required recalculating where the `caretBottom` when the flyout is up.  Screenshots are updated above.  Let me know if anything else seems amiss.